src/Impostor.Server/Net/Inner/Objects/InnerGameData.TaskInfo.cs

```diff
@@ -20,9 +20,9 @@ public void Serialize(IMessageWriter writer)
 
             public void Deserialize(IMessageReader reader)
             {
-                this.Type = (TaskTypes)reader.ReadPackedUInt32();
+                reader.ReadPackedUInt32();
```

Comment:
Maybe the index should be saved as well in case the plugin maker wants to do something with it.

Reply:
Index is already saved as the actual index in the List<TaskInfo>. Should we create another property for task index in TaskInfo?

Reply:
Yes, so it can be used in events where only an ITaskInfo is provided.